frontend/packages/client/src/components/Title.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Title = ({ children, className }) => (
```

Comment:
I think this one needs to be initialized with `''` otherwise when nothing is passed it injects undefined in the class prop:
```
<h6 class="small-text has-text-weight-bold is-uppercase mb-1 undefined">About</h6>